doc/source/ray-contribute/development.rst

```diff
@@ -343,9 +343,15 @@ You can tweak the build with the following environment variables (when running `
   python packages
 - ``RAY_DEBUG_BUILD``: Can be set to ``debug``, ``asan``, or ``tsan``. Any
   other value will be ignored
-- ``BAZEL_LIMIT_CPUS``: If set, it must be an integers. This will be fed to the
+- ``BAZEL_LIMIT_CPUS``: If set, this will be fed to the
   ``--local_cpu_resources`` argument for the call to Bazel, which will limit the
   number of CPUs used during Bazel steps.
+- ``BAZEL_LIMIT_RAM``: If set, this will be fed to the
+  ``--local_ram_resources`` argument for the call to Bazel, which will limit the
+  amount of RAM used during Bazel steps.
+- ``BAZEL_JOBS``: If set, this must be an integer. This will be fed to the
+  ``--jobs`` argument for the call to Bazel, which will limit the
+  number of parallel build jobs used during Bazel steps.
```

Comment:
just add a `BAZEL_ARGS` so that user can pass arbitrary flags (and deprecate `BAZEL_LIMIT_CPUS`)

Reply:
@aslonnie Totally happy with that idea, so I've done it. Thanks!